src/main/external-resources/renderers/Philips-PFL.conf

```diff
@@ -144,8 +138,6 @@ RendererIcon = philips-pfl.png
 UserAgentSearch = PhilipsIntelSDK/1.4|Streamium/1.0
 UpnpDetailsSearch = Philips TV
 
-TranscodeAudio = WAV
-MimeTypesChanges = audio/x-m4a = audio/mp4
```

Comment:
Although the mime `MimeTypesChanges = audio/x-m4a = audio/mp4` setting is not needed anymore because `MediaInfo = true` and the right mime type is given in the supported line, this is a renderer specific setting that is needed when using `StreamExtensions =`